src/content/whats-new/2024/02/whats-new-02-27-compute-add-on-sku

```diff
@@ -0,0 +1,20 @@
+---
+title: 'Launching the Compute Add On SKU'
+summary: 'Unlock access to all new capabilities, including Live Archives and IAST, with a single SKU'
+releaseDate: '2024-02-27'
+learnMoreLink: 'https://docs.newrelic.com/docs/licenses/license-information/usage-plans/new-relic-usage-plan/#list-price'
+---
+
+The Compute Add On enables you to configure and use all of New Relic’s newest innovations, including our recently-launched Live Archives and IAST, while remaining on the user model.
+
+You can now adopt our newest capabilities metered in Compute Capacity Units (CCU) without needing to migrate to New Relic Compute, and benefit from:
+
+* **Flexibility**: Optionality to stay on the data and user model and get access to new use cases via a single SKU.
+* **Metering consistency**: The same CCU metering applies to all new capabilities, including tracking, billing, invoicing, and reporting.
+* **Governance**: The same Compute Management UI is available when you purchase the add-on, enabling you to understand, monitor, and control your usage.
+* **Simplicity**: The add-on removes the complexity around purchasing separate SKUs, and if you are on a multi-year data and user contract, you can access net new use cases via a single amendment.
```

Comment:
@KarlTheFogNR the Compute Add On SKU only simplifies the billing for users of new features (for example, IAST and live archives). For users on Usage-based Pricing, they still need to purchase other add-ons for older features, don't they?

Reply:
@rhetoric101 Correct. In both cases, it's a simplification of the purchasing process.